examples/common/chip-app-server/QRCodeUtil.cpp

```diff
@@ -33,6 +37,14 @@ void PrintQRCode(chip::RendezvousInformationFlags rendezvousFlags)
     {
         ChipLogProgress(AppServer, "SetupPINCode: [%" PRIu32 "]", setupPinCode);
         ChipLogProgress(AppServer, "SetupQRCode:  [%s]", QRCode.c_str());
+
+        std::vector<char> qrCodeVector(3 * QRCode.size() + 1);
```

Comment:
Doesn't it increase FLASH size significantly? You may test if ScopedMemoryBuffer wouldn't result in smaller code.

Reply:
I tried using ScopedMemoryBuffer and it lowered FLASH usage about 80 B, so the difference is pretty significant. That was good idea, thanks.